context/app/static/js/components/entity-search/sidebar/DatasetConfigureSearchTable/utils.spec.js

```diff
@@ -0,0 +1,14 @@
+import { invertKeyToArrayMap } from './utils';
+
+test('should invert map', () => {
+  const initialMap = {
+    ablation_frequency_value: ['3D Imaging Mass Cytometry', 'Imaging Mass Cytometry'],
```

Comment:
```suggestion
    empty_value_ok: [],
    ablation_frequency_value: ['3D Imaging Mass Cytometry', 'Imaging Mass Cytometry'],
```
I don't know that this is actually a scenario you need to worry about, but it's an edge case.

Reply:
Where the logic has nothing to do with the particular names, slight preference to just use `'A', 'B', 'C'`: Less to be distracted by,